licenses.yaml

```diff
@@ -2872,6 +2872,18 @@ libraries:
   - io.confluent: kafka-schema-registry-client
   - io.confluent: common-utils
 
+---
+
+name: org.yaml snakeyaml
+license_category: binary
+module: extensions/druid-kubernetes-extensions
+license_name: Apache License version 2.0
+version: 2.0
+libraries:
+  - org.yaml: snakeyaml
+
+
+
```

Comment:
do we need this section since you already updated the version in the section above?

Reply:
fixed it to protobuf-extensions